<?xml version="1.0"?>
<launch>
    <arg name="markerId"        default="100"/>
    <arg name="markerSize"      default="0.075"/>    <!-- in m -->
	<arg name="camera_frame"    default="camera_rgb_optical_frame"/>
    <arg name="marker_frame"    default="head_aruco_marker_frame"/>
    <arg name="ref_frame"       default="camera_rgb_optical_frame"/>  <!-- leave empty and the pose will be published wrt param parent_name -->
    <arg name="corner_refinement" default="LINES" /> <!-- NONE, HARRIS, LINES, SUBPIX -->

   <node pkg="aruco_ros" type="single" name="aruco_single_head">
        <!--remap from="/camera_info" to="/pano_camera/rgb/camera_info" />
        <remap from="/image" to="/pano_camera/image_color" /-->
        <remap from="/camera_info" to="/camera/rgb/camera_info"/>
        <remap from="/image" to="/camera/rgb/image_rect_color"/>

		<param name="image_is_rectified" value="True"/>
        <param name="marker_size"        value="$(arg markerSize)"/>
        <param name="marker_id"          value="$(arg markerId)"/>
        <param name="reference_frame"    value="$(arg ref_frame)"/>   <!-- frame in which the marker pose will be refered -->
        <param name="camera_frame"       value="$(arg camera_frame)"/>
        <param name="marker_frame"       value="$(arg marker_frame)" />
        <param name="corner_refinement"  value="$(arg corner_refinement)" />
    </node>
</launch>

